src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -288,12 +288,25 @@ def build_output_dir(output_dir: Path, subdir: Optional[Union[str, Path]] = None
     return output_dir
 
 
-def build_final_results_directory(results_dir: str) -> Tuple[Path, Path]:
+def build_final_results_directory(
+    results_dir: str,
+    version: Optional[str] = None,
+) -> Tuple[Path, Path]:
+    if version is None:
```

Comment:
I think you should add a check that this version doesn't already exist and, if so, raise an error with an informative message.

Reply:
See other comment but this is making a new directory.